near/kycdao-ntnft/src/price_feed.rs

```diff
@@ -0,0 +1,18 @@
+use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
+use near_sdk::AccountId;
+
+#[derive(BorshDeserialize, BorshSerialize)]
+pub struct PriceFeed {
+    address: AccountId,
+}
+
+impl PriceFeed {
+    pub fn new(address: AccountId) -> PriceFeed {
+        PriceFeed { address }
+    }
+
+    pub fn last_price(&self) -> (u32, u8) {
+        //unimplemented!("TODO")
+        (17240, 4)
```

Comment:
Do we have a way to change the returned value easily? Do we need to deploy an updated version of the function? Is it easier to have a 'setter' method to call?

Reply:
I have the price change feature in a separate PR because it also requires data migration, so I just wanted to get this deployed on mainnet before pushing that feature. Also it might not be needed at all if I'll manage to finish the proper PriceFeed integration